src/zero/api/routes.clj

```diff
@@ -52,15 +52,10 @@
             :parameters {:query {:environment string?}}
             :responses {200 {:body {:total pos-int?}}}
             :handler handlers/status-counts}}]
-   ["/api/v1/wgs"
-    {:post {:summary    "Submit WGS Reprocessing workflows"
-            :parameters {:body ::spec/wgs-request}
-            :responses  {200 {:body {:results vector?}}}
-            :handler    handlers/submit-wgs}}]
```

Comment:
Can you also remove the `submit-wgs` handler? I don't think anything else is using it

Reply:
Good catch! Removed.